WordPress/Resources/release_notes.txt

```diff
@@ -1,12 +1,20 @@
-What time is it? Our brand-new time zone selection screen knows. Get a suggested time zone based on your device and improve search while you’re at it.
+* [*] Signup: Fixed bug where username selection screen could be pushed twice. [#17624]
+* [**] Threaded comments: comments can now be moderated via a drop-down menu on each comment. [#17758]
+* [**] Reader post details Comments snippet: added ability to manage conversation subscription and notifications. [#17749]
```

Comment:
Notice that this item regarding the Reader is only part of the WordPress release notes 👀